client/my-sites/email/form/mailboxes/index.ts

```diff
@@ -135,11 +183,11 @@ class MailboxForm< T extends EmailProvider > {
 		}
 	}
 
-	validate() {
+	validate( skipInvisibleFields = false ) {
 		this.clearErrors();
 
 		for ( const [ fieldName, validator ] of this.getValidators() ) {
-			if ( ! fieldName ) {
+			if ( ! fieldName || ! validator ) {
```

Comment:
Looking at the implementation of `getValidators`, it doesn't look like there is any real risk of `validator` being `undefined`. Am I missing something here?